networkit/cpp/io/DibapGraphReader.cpp

```diff
@@ -5,17 +5,34 @@
  *      Author: Henning
  */
 
-#if !defined _WIN32 && !defined _WIN64 && !defined WIN32 && !defined WIN64
+#ifndef DNETWORKIT_WINDOWS
```

Comment:
AFAICS the only reason that this is not available on Windows is that we need `htonX`? This is not something that should be changed in this PR but in the future we could migrate to an internal (or TLX?) byteswapping function.